test_data/asm/poseidon_bn254.asm

```diff
@@ -1,11 +1,15 @@
+machine Poseidon(LASTBLOCK, function_id) {
+
+    degree 1024;
```

Comment:
If you remove the degree it should just be either copied from Main or assigned 1024 automatically by the linker I think

Reply:
https://powdr-labs.github.io/powdr/linker/index.html